vips/vip-170/vip-170-testnet.ts

```diff
@@ -0,0 +1,185 @@
+import { ethers } from "hardhat";
+
+import { ProposalType } from "../../src/types";
+import { makeProposal } from "../../src/utils";
+
+const VTOKEN_BEACON = "0xBF85A90673E61956f8c79b9150BAB7893b791bDd";
+const NEW_IMPL_VTOKEN = "0x37130dd8181477Be3dDe8b22A32FE302ca602BA7";
```

Comment:
This code is not the last one (according to https://github.com/VenusProtocol/isolated-pools/pull/207). I would deploy the last version of the code

Reply:
Updated impl addr [af09fb0](https://github.com/VenusProtocol/vips/pull/67/commits/af09fb00273c1ff846a5c74d45c6c17b0d6216b9)